README.md

```diff
@@ -101,6 +101,16 @@ However, in the case that it needs to be disabled, locate the RTCConfiguration p
 PeerConnection.RTCConfiguration rtcConfig = new PeerConnection.RTCConfiguration();
 rtcConfig.continualGatheringPolicy = PeerConnection.ContinualGatheringPolicy.GATHER_CONTINUALLY;
 ```
+## 7. Testing
+This SDK has been tested with Java 11, 17 to build the Gradle dependencies and Java 8, 11, and, 17 in the compile options in build.gradle
```

Comment:
8 is mentioned but not in the CI

Reply:
Those two are different. One is to build gradle dependencies (Java 11 and 17 for gradle 7.x) and the other is to compile and build the source (Java 8, 11, 17)
https://developer.android.com/build/jdks#jdk-config-in-studio